### ISSUE TITLE
An error occurred when use_sockets set to false

### ISSUE THREAD

**Issue by msjaber** [howdyai/botkit-starter-web#27]
Whenever `use_sockets` is `client.js` file is disabled, this error occurs.

`An error occurred while sending a message:  Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client`

**Comment by adantoscano** [comment]
You are trying to send two replies or convos through a webhook

**Comment by msjaber** [comment]
@adantoscano Hmm... I only changed `use_socket` to `false` and didn't try to reply anything. Try to Remix it on Glitch and see.

**Comment by adantoscano** [comment]
Yup but if you keep socket as true, you can see that you receive two messages. Changing socket to false you're using webhook and with it cannot perform more than a replay each response
